connect/mirror/src/main/java/org/apache/kafka/connect/mirror/ConfigPropertyFilter.java

```diff
@@ -25,8 +25,19 @@
 @InterfaceStability.Evolving
 public interface ConfigPropertyFilter extends Configurable, AutoCloseable {
 
+    /**
+     * Specifies whether to replicate the given topic configuration whether it has a default value or not.
+     * The configurations with default values are controlled by shouldReplicateSourceDefault.
```

Comment:
1. This still needs to be updated if we make `shouldReplicateConfigProperty` and `shouldReplicateSourceDefault` handle distinct properties
2. We can tidy up the Javadocs a bit by including a link to the relevant method

```suggestion
     * Specifies whether to replicate the given topic configuration whether it has a default value or not.
     * The configurations with default values are controlled by {@link #shouldReplicateSourceDefault(String)}. 
```